helpers/helper-translation-discussion.php

```diff
@@ -190,12 +190,37 @@ function() {
 				'translation_id'     => isset( $this->data['translation_id'] ) ? $this->data['translation_id'] : null,
 				'locale_slug'        => $this->data['locale_slug'],
 				'original_permalink' => $this->data['permalink'],
+				'languages'          => $this->generateCommentFilterLinks( $comments, $this->data['locale_slug'] ),
 			),
 			$this->assets_dir . 'templates'
 		);
 		return $output;
 	}
 
+	public function generateCommentFilterLinks( $comments, $locale_slug ) {
```

Comment:
I think the convention for function names is snake_case.

Reply:
Oops!